flowz/artifacts.py

```diff
@@ -14,7 +14,7 @@ class AbstractArtifact(object):
     An object that wraps the details for asynchronous access to an artifact.
     """
 
-    logger = logging.getLogger('Artifact')
+    logger = logging.getLogger(__name__)
```

Comment:
I don't think this use of `__name__` means what you think.

It isn't resolving to the name of the class; it resolves to the name of the module.